frontend/apps/staff/.env.example

```
@@ -3,6 +3,14 @@ NEXT_PUBLIC_BASE_URL=/kasittely
 RESERVATION_UNIT_PREVIEW_URL_PREFIX="http://localhost:3000/reservation-unit"
 FEEDBACK_URL=
-SENTRY_DSN=
+SENTRY_ORG=city-of-helsinki
+SENTRY_PROJECT=tilavarauspalvelu-staff-ui
+NEXT_PUBLIC_SENTRY_PROJECT=${SENTRY_PROJECT}
 SENTRY_AUTH_TOKEN=
-SENTRY_ENVIRONMENT=local
+SENTRY_ENABLE_SOURCE_MAPS=false
+NEXT_PUBLIC_SENTRY_TRACE_PROPAGATION_TARGETS=http://localhost:8000
+NEXT_PUBLIC_SENTRY_DSN=
+NEXT_PUBLIC_SENTRY_ENVIRONMENT="local"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)